### PR TITLE
HTML-prototype

### PR DESCRIPTION
**Pull request by akimov228aleksei**
Add HTML-prototype

### CODE REVIEW

**Review by jimmyhollywood — CHANGES_REQUESTED** [review]
I think it would be nice to add pages for "add" and "delete" actions. Just to show how the features will work.

**Comment by akimov228aleksei** [comment]
Removed the HTML prefix from the file names, made the "pictures" folder in lowercase.Fixed paths of pictures

File: Documentations/HTML_prototype/departments_after_edit.html [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        <td style="border-bottom-color:red; border-bottom-width:2; border-left-width:0; border-right-width:0; text-align:center;">
+            <a href="departments_edit.html"><img src="pictures/pan.png" width="30" height="50" alt="."></a></td>
+        <td style="border-left-width:0; border-bottom-color:red; border-bottom-width:2; text-align:right;"><img
+                src="pictures/trash.png" width="30" height="50" alt="."></td>
+    </tr>
+
+    <tr>
+        <td style="cursor:pointer" onclick="document.location='vacation.html'" bgcolor="DarkGray" align="center" rowspan="2"><font size="30">Vacation</font></td>
+        <td style="border-bottom-color:red; border-bottom-width:2; color:red">Oncology</td>
+        <td style="border-bottom-color:red; border-bottom-width:2;"> 3213 $</td>
```
Comment (jimmyhollywood) [inline Documentations/HTML_prototype/departments_after_edit.html:75]: Just wondering, why the average salary is changed after editing of the department name?

**Review by jimmyhollywood — CHANGES_REQUESTED** [review]
Looks good, just one suggestions. When add or edit record set new name like a "Edited/Added Department/Person/Vacation"

**Comment by akimov228aleksei** [comment]
Created missing templates. Added actions for buttons "edit" and "delete".

File: Documentations/HTML_prototype/departments_after_edit.html (hunk 1 above)
Comment (akimov228aleksei) [reply Documentations/HTML_prototype/departments_after_edit.html]: Fixed.

**Comment by akimov228aleksei** [comment]
> Выглядит хорошо, только одно предложение. При добавлении или редактировании записи установите новое имя, например «Отредактировано / Добавлен отдел / Человек / Отпуск».

Fixed.

**Review by jimmyhollywood — APPROVED** [review]
That is ok.
_Note: When you will work with html pages for your web application - try to use separate css files_

**Pull request merged**